Taweret/utils/utils.py

```diff
@@ -50,11 +50,25 @@ def mixture_function(method : str, x : np.ndarray, mixture_params : np.ndarray)
     if method=='sigmoid':
         theta_0, theta_1 = mixture_params
         w = expit((x-theta_0)/theta_1)
+        return w, 1 - w
```

Comment:
All return objects from this function should have the same type. Either a list of numbers or an numpy array. I think NumPy array would be better.

Reply:
Done